Source/Core/Core/HW/EXI/EXI_DeviceEthernet.cpp

```diff
@@ -43,6 +37,39 @@ CEXIETHERNET::CEXIETHERNET()
     SConfig::GetInstance().SaveSettings();
   }
 
+  switch (type)
+  {
+  case BBADeviceType::TAP:
+    m_network_interface = std::make_unique<TAPNetworkInterface>(this);
+    INFO_LOG(SP1, "Created TAP physical network interface.");
+    break;
+  case BBADeviceType::XLINK:
+    // Perform sanity check on BBA MAC address, XLink requires the vendor OUI to be correct.
+    // Don't actually stop the BBA from initializing though
+    if (mac_addr_setting.substr(0, 8) != "00:09:bf" && mac_addr_setting.substr(0, 8) != "00:17:ab")
+    {
+      NOTICE_LOG(SP1,
```

Comment:
Maybe this should also be a PanicAlertT, since XLink will refuse the bad MAC address. Otherwise, the user might not understand what's going on.

Reply:
Agreed, no point in continuing if the BBA wont work